src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/DisputeService.java

```diff
@@ -221,4 +222,16 @@ public void unassignDisputes() {
 		logger.debug("Unassigned {} record(s)", count);
 	}
 
+	/**
+	 * Flips the Dispute.emailAddressVerified flag to true where Dispute.emailVerificationToken matches the given parameter
+	 * @param token the Dispute record to update
+	 */
+	public void validateEmail(String emailVerificationToken) {
+		List<Dispute> emailVerificationTokens = disputeRepository.findByEmailVerificationToken(emailVerificationToken);
```

Comment:
will the C# api method return the dispute id or other information needed by the front end to redirect the user to submission confirmation?

Reply:
and will c# methods be part of another PR?

Reply:
Another PR for the c# stuff, yes. I try hard to keep PRs under 6 files or so to keep reviews simple.

The C# PR is unable to return the dispute ID since the Dispute hasn't been saved yet (the workflow service does that later).

So the c# PR (working on now) will return the emailValidationToken to the UI when first submitting the Dispute. This way the user can click a button to resend the email validation. This same token (uuid) is persisted to the database for later lookup by this api so we can send email validation on a whim.